python/pyspark/pandas/tests/test_dataframe_conversion.py

```diff
@@ -101,69 +101,69 @@ def test_to_excel(self):
             koalas_location = dirpath + "/" + "output2.xlsx"
```

Comment:
Shall we rename `koalas_location`? Please ignore if it's out of the scope of this PR.

Reply:
Actually there are a few more places.
- `koalas_result` in `test_series.py`.
- a test name `test_as_spark_type_koalas_dtype` in `test_typedef.py`

Reply:
Thanks!! Just addressed them